src/avfoundation.cs

```diff
@@ -315,10 +423,90 @@ interface AVMediaCharacteristic {
 		[Field ("AVMediaCharacteristicVoiceOverTranslation")]
 		NSString VoiceOverTranslation { get; }
 	}
+#endif
+
+	[NoWatch]
+	enum AVMetadataFormat {
+		[iOS (8,0)][Mac (10,10)]
+		[Field ("AVMetadataFormatHLSMetadata")]
+		FormatHlsMetadata = 0,
+
+		[Field ("AVMetadataFormatiTunesMetadata")]
+		FormatiTunesMetadata = 1,
+
+		[Field ("AVMetadataFormatID3Metadata")]
+		FormatID3Metadata = 2,
```

Comment:
in this case we already have KeySpaceID3, so I'm keeping the casing unless we of to set KeySpaceID3 [Obsolete], rename it and the use FormatID3Metadata.